pages/common/pulumi.md

```diff
@@ -0,0 +1,28 @@
+# pulumi
+
+> Define infrastructure on any cloud using familiar programming languages.
+> More information: <https://www.pulumi.com/docs/reference/cli>.
+
+- Create a new project using a template:
+
+`pulumi new`
+
+- Create a new stack, an isolated deployment target:
+
+`pulumi stack init`
+
+- Interactively configure variables such as keys, regions, etc:
+
+`pulumi config`
+
+- Preview and deploy changes to a program and/or infrastructure:
```

Comment:
This command has a confirmation prompt?

Reply:
That command lists all configuration values for a specific stack. 
To add a new configuration value  `pulumi config set`. 
To remove an existing value run `pulumi config rm`. 
To get the value of for a specific configuration key, use `pulumi config get`